docs/application/dotnet/guides/nui/visuals.md

```diff
@@ -1,638 +1,772 @@
+   _meshVisual.MaterialtURL = _imageUrl + "Dino.mtl";
+   _meshVisual.TexturesPath = _imageUrl + "textures/";
 
-The animated image visual renders an animated image into the control's quad. Currently, only the GIF format is supported, but an API to enable multiple images to be displayed sequentially is being developed.
+   _visualView.AddVisual("Mesh-Image", _meshVisual);
+   ```
 
-**Figure: Animated image visual**
+The final output after implementing the preceding code is as follows:
 
-![Animated image visual](media/animated-image-visual.gif)
+![Mesh visual](media/visuals/MeshVisualExample.png)
 
-The visual map for an animated image is `AnimatedImageVisual`.
+## PrimitiveVisual
 
-<a name="meshvisual"></a>
-## Mesh Visual
+The `PrimitiveVisual` renders a simple three-dimensional shape, such as cube or sphere. By default, shapes are generated with clockwise winding and back-face culling.
 
-The mesh visual renders a mesh using a `.obj` file, optionally with textures provided by a `.mtl` file. The mesh is scaled to fit the control.
+> [!NOTE]
+> Due to a typographical mistake in the source code, it is advised to use 'ConicalFrustrum' instead of 'ConicalFrustum' as one of the 'Shape' names. 
 
-**Figure: Mesh visual**
+The following table lists the supported properties:
 
-![Mesh visual](media/mesh-visual.png)
+**Table: The optional properties of the PrimitiveVisual.**
 
-The following table lists the supported properties. The visual type is `Mesh` and the visual map is `MeshVisual`.
+| Property            | Type                       | Shape to which the property applies  | Description                                                                                                        |
+|---------------------|----------------------------|--------------------------------------|--------------------------------------------------------------------------------------------------------------------|
+| `Shape`             | `PrimitiveVisualShapeType` | all                                  | The shape to render. The default value is `Sphere`. For more information, see the [table](#table_primit_possible_values). |
+| `MixColor`          | `Color`                    | all                                  | The color of the shape. The default value is `[0.5f, 0.5f, 0.5f, 1.0f]`.                                           |
+| `Slices`            | `int` in a range 1-255     | all                                  | The number of slices going around the shape. The default value is `128`.                                           |
+| `Stacks`            | `int`                      | all                                  | The number of layers going down the shape. The default value is `128`.                                             |
+| `ScaleTopRadius`    | `float` greater than or equal to 0 | `ConicalFrustrum`                         | The scale of the top circle radius of a conical frustum. The default value is `1.0`.                  |
```

Comment:
| `ScaleTopRadius`    | `float` greater than or equal to 0 | `ConicalFrustrum`                         | The scale of the top circle radius of a conical frustum. The default value is `1.0`.                |

Can you rephrase: The scale of the top circle radius of a conical frustum. The default value is `1.0`. 
Also for : 627,628

Reply:
Can you specify the reason why it should be rephrased? I think it sounds good so I don't know which part of these sentences to rephrase

Reply:
@mgrudzinska 
 The scale of the top circle radius of a conical frustum. The default value is 1.0.

This sentence needs to be rephrased, this is not clear. Are these two sentences?

Reply:
I changed as follows:

The scale of the radius of the top circle of the conical frustum. The default value is `1.0`.

The scale of the radius of the bottom circle of the conic. The default value is `1.5`. 

The scale of the height of the conic. The default value is `3.0`. 

The scale of the radius of the cylinder. The default value is `1.0`.